tracker/tracker-assist/src/CallWindow.ts

```diff
@@ -327,6 +330,10 @@ export default class CallWindow {
 		this.adjustIframeSize()
 	}
 
+	public setVideoCallback(cb) {
```

Comment:
onVideoToggle() or setVideoToggleCallback() or something else with the notion of what it actually is